js/customize-snapshots.js

```diff
@@ -273,8 +299,216 @@
 		 *
 		 * @returns {string} URL.
 		 */
-		getSnapshotFrontendPreviewUrl: function() {
+		getSnapshotFrontendPreviewUrl: function getSnapshotFrontendPreviewUrl() {
 			return api.previewer.getFrontendPreviewUrl();
+		},
+
+		/**
+		 * Resets conflicts.
+		 *
+		 * @param {boolean} saved - If saved or not.
+		 * @return {void}
+		 */
+		resetConflicts: function resetConflicts( saved ) {
+			var snapshot = this;
+
+			if ( ! saved ) {
+				return;
+			}
+
+			_.each( snapshot.conflict.controlThickboxes, function( thickBox ) {
+				thickBox.remove();
+			} );
+
+			if ( snapshot.conflict.currentRequest ) {
+				snapshot.conflict.currentRequest.abort();
+				snapshot.conflict.currentRequest = null;
+			}
+
+			snapshot.conflict.controlThickboxes = {};
+			snapshot.conflict.controlsWithPendingRequest = {};
+
+			api.control.each( function( control ) {
+				_.each( control.settings, function( setting ) {
+					setting.unbind( snapshot.handleConflictRequestOnFirstChange );
+				} );
+				control.notifications.remove( snapshot.conflict.notificationCode );
+				snapshot.addConflictButton( control );
+			} );
+		},
+
+		/**
+		 * Handle conflict on first settings change.
+		 *
+		 * @param {wp.customize.Control} control - Control.
+		 * @return {void}
+		 */
+		handleConflictRequestOnFirstChange: function handleConflictRequestOnFirstChange( control ) {
+			var snapshot = this;
+
+			_.each( control.settings, function( setting ) {
+				setting.unbind( snapshot.handleConflictRequestOnFirstChange );
```

Comment:
Per below, I don't think this is going to work. There are no calls to `setting.bind( snapshot.handleConflictRequestOnFirstChange )` to unbind.